cirq/experiments/n_qubit_tomography.py

```diff
@@ -45,7 +45,7 @@ class StateTomographyExperiment:
     """
 
     def __init__(self,
-                 qubits: Sequence[cirq.Qid],
+                 qubits: Sequence[ops.Qid],
```

Comment:
I think the current consensus for annotations is to add
```python
if TYPE_CHECKING:
    import cirq
```
and use `'cirq.Qid'` so the documentation is consistent.

Reply:
Good point. Done.